src/BrightScriptDebugSession.ts

```diff
@@ -158,7 +190,11 @@ export class BrightScriptDebugSession extends DebugSession {
             }
         } catch (e) {
             console.log(e);
-            this.sendErrorResponse(response, -1, e.message);
+            if (e.message !== 'compileErrors') {
+                this.sendErrorResponse(response, -1, e.message);
+            } else {
+                //TODO make the debugger stop!
```

Comment:
Is this something you plan on coding before completing this PR?

Reply:
no. I don't know how to do it. This TODO is for someone else - or a future self with more time.